Formula/libpipeline.rb

```diff
@@ -0,0 +1,31 @@
+class Libpipeline < Formula
+  desc "C library for manipulating pipelines of subprocesses"
+  homepage "http://libpipeline.nongnu.org/"
+  url "https://download.savannah.nongnu.org/releases/libpipeline/libpipeline-1.5.1.tar.gz"
+  sha256 "d633706b7d845f08b42bc66ddbe845d57e726bf89298e2cee29f09577e2f902f"
+
+  depends_on "pkg-config"
+
+  def install
+    ENV["CC"] = "#{ENV.cc} -arch x86_64"
```

Comment:
As far as I know the homebrew env already defines this, can you check without?

Reply:
Removed.